doc/man1/openssl-namefmt-options.pod

```diff
@@ -0,0 +1,179 @@
+=pod
+
+=head1 NAME
+
+openssl-namefmt-options - DN display options
```

Comment:
"Name display options" or "Distinguished name display options"

Also, I'm wondering if these shouldn't be exactly that throughout the manuals, "name display options".  That would avoid having them mixed up with the other format options

Reply:
That includes the file name of this pod.